contracts/interfaces/IStakingPool.sol

```diff
@@ -8,7 +8,7 @@ import "@openzeppelin/contracts-v4/token/ERC721/IERC721.sol";
 
 struct WithdrawParams {
   uint positionId;
-  uint[] groupIds;
+  uint groupId;
   uint flags;
```

Comment:
Instead of `flags` we could just use two booleans.

Reply:
Why is groupIds no longer an array? I'm using this in token controller to withdraw all rewards in one call. This would force me to make a call for every term for every nft for every pool.